src/Microphysics2M.jl

```diff
@@ -445,27 +446,34 @@ function rain_evaporation(
     end
 
     return (; evap_rate_0, evap_rate_1)
+    return (; evap_rate_0, evap_rate_1)
 end
 
-"""
-    radar_reflectivity(struct, q_liq, q_rai, N_liq, N_rai, ρ_air, ρ_w, τ_q, τ_N)
+""" 
+    radar_reflectivity(structs, q_liq, q_rai, N_liq, N_rai, ρ_air, ρ_w, τ_q, τ_N)
 
-    - `struct` - type for 2-moment rain autoconversion parameterization
-    - `q_liq` - cloud water specific humidity
+    - `structs` - structs with SB2006 cloud droplets and raindrops 
+                size distributions parameters
+    - `q_liq` - cloud water specific humidity 
     - `q_rai` - rain water specific humidity
     - `N_liq` - cloud droplet number density
     - `N_rai` - rain droplet number density
+    - `N_liq` - cloud droplet number density
```

Comment:
Looks like during rebasing we accepted both lines. We only need one `N_liq` etc :)